docs/reference/mobile/prebuilt-package/1.9 ORTMobilePackageOperatorTypeSupport.md

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
Can we change this file (and `1.8 ORTMobile...` as well) to not use space in the file name?